### PR TITLE
add meds and des-desmeds recipes

### PR DESCRIPTION
**Pull request by beckermr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/meds/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  license_family: GPL
+  license_file: LICENSE
+  summary: Python and C libraries to work with Multi Epoch Data Structures
+  doc_url: https://github.com/esheldon/meds
+  dev_url: https://github.com/esheldon/meds
+
+extra:
+  recipe-maintainers:
+    - beckermr
+    - esheldon
```
Comment (beckermr) [inline recipes/meds/meta.yaml]: @esheldon, would you like to help maintain this? We chatted offline but I did not ask.

File: recipes/des-desmeds/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  license_family: GPL
+  license_file: LICENSE
+  summary: DES-specific MEDS production code
+  doc_url: https://github.com/esheldon/desmeds
+  dev_url: https://github.com/esheldon/desmeds
+
+extra:
+  recipe-maintainers:
+    - beckermr
+    - esheldon
```
Comment (beckermr) [inline recipes/des-desmeds/meta.yaml]: This one too @esheldon. Feel free to pass on maintaining if you would like.

File: recipes/des-desmeds/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - desmeds
+
+about:
+  home: https://github.com/esheldon/desmeds
+  license: GPL-3
+  license_family: GPL
+  license_file: LICENSE
```
Comment (beckermr) [inline recipes/des-desmeds/meta.yaml]: We need to add the license to the repo and cut a new release for desmeds.

File: recipes/meds/meta.yaml (hunk 1 above)
Comment (esheldon) [reply recipes/meds/meta.yaml]: yep

File: recipes/des-desmeds/meta.yaml (hunk 2 above)
Comment (esheldon) [reply recipes/des-desmeds/meta.yaml]: yep

File: recipes/des-desmeds/meta.yaml (hunk 3 above)
Comment (esheldon) [reply recipes/des-desmeds/meta.yaml]: done

File: recipes/des-desmeds/meta.yaml (hunk 2 above)
Comment (beckermr) [reply recipes/des-desmeds/meta.yaml]: answered above

**Comment by beckermr** [comment]
@conda-forge/help-python These recipes are ready for review.

**Pull request merged**